test/studies/shootout/k-nucleotide/bharshbarg/PREEXEC

```diff
@@ -10,10 +10,10 @@ if [ -n "$CHPL_TEST_PERF" ] && [ ! -f $CHPL_TEST_TMP_DIR/fasta.big ]; then
   $3 $CHPL_HOME/test/studies/shootout/fasta/kbrady/fasta-printf.chpl --fast -o tmpFasta
   
   # needed by k-nucleotide and reverse-complement
-  ./tmpFasta --n=25000000 > $CHPL_TEST_TMP_DIR/fasta.big
+  ./tmpFasta --n=25000000 -nl 1 > $CHPL_TEST_TMP_DIR/fasta.big
```

Comment:
Is -nl 1 okay in --local compilations, or should this addition be guarded by CHPL_COMM != none? (I thought it wasn't, which is why I ask)